src/VisualStudio/Razor/project.json

```diff
@@ -0,0 +1,7 @@
+﻿{
+    "dependencies": {
+    },
+    "frameworks": {
+        "net46": { }
```

Comment:
I copied this from what I saw in the XamlVisualStudio and ServiceHub projects.  don't claim to understand what's going on here, I'm just doing the same thing.

https://github.com/dotnet/roslyn/blob/master/src/Workspaces/Remote/ServiceHub/project.json
https://github.com/dotnet/roslyn/blob/master/src/VisualStudio/Xaml/Impl/project.json

If you think I can/should delete this I can, I just don't understand the implications.